.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "automl_common"]
+	path = automl_common
```

Comment:
Okay, I see your point but I would also say that anyone who is installing with `pip` will never see any of this. Only github clone and install'ers will see it, in which case transparency that there is a subpackage might be nice.

On a more practical note, it also means the tests distributed with `automl_common` don't end up in the source code of autosklearn.

Reply:
Hey, I just had another look and I think there is another issue here: the `.tar.gz` doesn't include the submodule as it only packages everything under `autosklearn`. So I think that `automl_common` actually has to live under `autosklearn`. Test etc can be excluded with `exclude` in the [`MANIFEST.in`](https://packaging.python.org/guides/using-manifest-in/).

However, this raises the important issue that the CI wasn't strict enough. We need to go to a different directory to run the tests as [done in scikit-learn](https://github.com/scikit-learn/scikit-learn/blob/main/build_tools/github/test_source.sh#L17) to make sure we don't accidentally import what's in the current working directory.

Reply:
It can definitely live outside autosklearn and just build the package differently but sure, I don't mind either way,  we can exclude them that way.

* I will put `automl_common` under autosklearn.
* I will see how to test that `MANIFEST.in` works with excluding test and other files.
* I will take a look at the sklearn things

Reply:
* I put it underneath and it seems to work
* I updated the manifest to not include automl_commons tests
* Regarding testing, our test workflow already specifies to use the correct autosklearn test folder

Reply:
> Regarding testing, our test workflow already specifies to use the correct autosklearn test folder

Do we `cd` in some other directory? I can't find that right now. What I meant is that we execute the tests from for example `/tmp/tests` so that we don't by accident import anything from the current working directory.

Reply:
I don't see why that should now be an issue? We'll see what the tests say

Reply:
tests seem to run without problem and it only runs `autosklearn` tests